SQL/scripts.sql

```diff
@@ -0,0 +1,3 @@
+create schema logicdb;
```

Comment:
why you creating your own schema name and maintain across all people one schema name.

Reply:
maintained Schema